sw/airborne/firmwares/rotorcraft/autopilot_firmware.c

```diff
@@ -110,8 +110,12 @@ static void send_energy(struct transport_tx *trans, struct link_device *dev)
 {
   uint8_t throttle = 100 * autopilot.throttle / MAX_PPRZ;
   float power = electrical.vsupply * electrical.current;
+  float avg_power = (float)electrical.avg_power / electrical.avg_cnt;
```

Comment:
if counter is zero (see line below), you shouldn't divide

Reply:
Changed also added it to other firmwares